oap-server/server-configuration/configuration-zookeeper/src/main/java/org/apache/skywalking/oap/server/configuration/zookeeper/ZookeeperConfigWatcherRegister.java

```diff
@@ -46,9 +51,28 @@ public ZookeeperConfigWatcherRegister(ZookeeperServerSettings settings) throws E
     @Override
     public Optional<ConfigTable> readConfig(Set<String> keys) {
         ConfigTable table = new ConfigTable();
-        keys.forEach(s -> {
-            ChildData data = this.childrenCache.getCurrentData(this.prefix + s);
-            table.add(new ConfigTable.ConfigItem(s, data == null ? null : new String(data.getData())));
+        keys.forEach(key -> {
+            if (this.getRegister().get(key).getWatcher().getWatchType() == ConfigChangeWatcher.WatchType.GROUP) {
```

Comment:
I think about enum, we should use `ConfigChangeWatcher.WatchType.GROUP.equals`

Reply:
Make `this.getRegister().get(key).getWatcher().getWatchType()` a parent method please.